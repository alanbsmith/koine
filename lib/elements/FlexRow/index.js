import styled from "styled-components";
import { rem } from "polished";

const FlexRow = styled.div`
  box-sizing: border-box;
  display: flex;
  flex-wrap: wrap;
  flex: 1;
  padding: ${props => rem(props.theme.dimensions.basePad)};
`;

/** @component */
export default FlexRow;
