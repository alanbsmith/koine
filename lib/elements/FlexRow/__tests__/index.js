import React from 'react';
import FlexRow from '../index';
import renderer from 'react-test-renderer';


describe('FlexRow Element', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <FlexRow />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
