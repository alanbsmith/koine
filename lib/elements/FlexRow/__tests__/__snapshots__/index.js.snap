// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexRow Element renders correctly 1`] = `
<div
  className="sc-bdVaJa jAkyeT"
/>
`;
